src/components/Details/DetailsView.js

```diff
@@ -63,13 +66,33 @@ const DetailsView = React.forwardRef(
       selectedItem.state || {}
 
     return (
-      <div className={detailsPanelClassNames} ref={ref}>
+      <div className={detailsPanelClassNames} ref={ref} style={style}>
         {detailsStore.loading && <Loader />}
         {detailsStore.error && (
           <ErrorMessage message={detailsStore.error.message} />
         )}
         <div className="item-header__data">
           <h3 className="item-header__title">
+            {isDetailsScreen && (
+              <Link
+                className="item-header__back-btn"
+                to={location => {
+                  const urlArray = location.pathname.split('/')
+                  return urlArray.slice(0, urlArray.length - 2).join('/')
```

Comment:
`Array.prototype.slice(start, end)` could take a negative integers as its second argument (`end`) which counts backwards from the end of the array.
```suggestion
                  return urlArray.slice(0, -2).join('/')
```